disperser/batcher/batcher.go

```diff
@@ -193,6 +345,12 @@ func (b *Batcher) handleFailure(ctx context.Context, blobMetadatas []*disperser.
 	// Return the error(s)
 	return result.ErrorOrNil()
 }
+
+type confirmationMetadata struct {
+	batch  *batch
```

Comment:
It looks like what we need here are the various metadata objects associated with the batch; specifically, we don't need the encoded blobs. 

More conserving memory, is it worth excluding the encoded blobs? Either by
- Deleting them from the object, or
- More explicitly (so maybe better), expanding the confirmationMetadata object to directly include the batch fields that we need to preserve

Reply:
Sounds good. I'll specify fields more granularly